www-misc/zoneminder/zoneminder-1.34.8.ebuild

```diff
@@ -0,0 +1,178 @@
+	dev-perl/libwww-perl
+	dev-php/pecl-apcu:*
+	sys-libs/zlib
+	virtual/ffmpeg
+	virtual/httpd-php:*
+	virtual/jpeg:0
+	virtual/perl-ExtUtils-MakeMaker
+	virtual/perl-Getopt-Long
+	virtual/perl-Sys-Syslog
+	virtual/perl-Time-HiRes
+	www-servers/apache
+	curl? ( net-misc/curl )
+	gcrypt? ( dev-libs/libgcrypt:0= )
+	gnutls? ( net-libs/gnutls )
+	mmap? ( dev-perl/Sys-Mmap )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	vlc? ( media-video/vlc[live] )
+"
+RDEPEND="${DEPEND}"
+
+# we cannot use need_httpd_cgi here, since we need to setup permissions for the
+# webserver in global scope (/etc/zm.conf etc), so we hardcode apache here.
+need_apache
+
+PATCHES=(
+	"${FILESDIR}/${PN}"-1.34-systemd.patch
+)
+
+MY_ZM_WEBDIR=/usr/share/zoneminder/www
+
+src_prepare() {
+	cmake-utils_src_prepare
+	rmdir "${S}/web/api/app/Plugin/Crud" || die
+	mv "${WORKDIR}/crud-${MY_CRUD_VERSION}" "${S}/web/api/app/Plugin/Crud" || die
+	rmdir "${S}/web/api/app/Plugin/CakePHP-Enum-Behavior" || die
+	mv "${WORKDIR}/CakePHP-Enum-Behavior-${MY_CAKEPHP_VERSION}" "${S}/web/api/app/Plugin/CakePHP-Enum-Behavior" || die
+}
+
+src_configure() {
+	append-cxxflags -D__STDC_CONSTANT_MACROS
+	perl_set_version
+	export TZ=UTC # bug 630470
+	mycmakeargs=(
+		-DZM_CONFIG_DIR=/etc/zm
+		-DZM_NO_CURL="$(usex curl OFF ON)"
+		-DZM_NO_LIBVLC="$(usex vlc OFF ON)"
+		-DZM_NO_MMAP="$(usex mmap OFF ON)"
+		-DZM_NO_X10=OFF
+		-DZM_PERL_SUBPREFIX=${VENDOR_LIB#/usr}
+		-DZM_SOCKDIR=/var/run/zm
+		-DZM_TMPDIR=/var/tmp/zm
+		-DZM_WEBDIR=${MY_ZM_WEBDIR}
+		-DZM_WEB_GROUP=apache
+		-DZM_WEB_USER=apache
+		-DCMAKE_DISABLE_FIND_PACKAGE_OpenSSL="$(usex ssl OFF ON)"
+		-DHAVE_GNUTLS="$(usex gnutls ON OFF)"
+		-DHAVE_GCRYPT="$(usex gcrypt ON OFF)"
+	)
+
+	cmake-utils_src_configure
+
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	# the log directory
+	keepdir /var/log/zm
+	fowners apache:apache /var/log/zm
+
+	# the logrotate script
+	insinto /etc/logrotate.d
+	newins distros/ubuntu1204/zoneminder.logrotate zoneminder
+
+	# now we duplicate the work of zmlinkcontent.sh
+	keepdir /var/lib/zoneminder /var/lib/zoneminder/images /var/lib/zoneminder/events /var/lib/zoneminder/api_tmp
+	fperms -R 0775 /var/lib/zoneminder
+	fowners -R apache:apache /var/lib/zoneminder
+	dosym var/lib/zoneminder/images ${MY_ZM_WEBDIR}/images
+	dosym var/lib/zoneminder/events ${MY_ZM_WEBDIR}/events
+	dosym var/lib/zoneminder/api_tmp ${MY_ZM_WEBDIR}/api/app/tmp
+
+	# bug 523058
+	keepdir ${MY_ZM_WEBDIR}/temp
+	fowners -R apache:apache ${MY_ZM_WEBDIR}/temp
+
+	# the configuration file
+	fperms 0640 /etc/zm/zm.conf
+	fowners root:apache /etc/zm/zm.conf
+
+	# init scripts etc
+	newinitd "${FILESDIR}"/init.d zoneminder
+	newconfd "${FILESDIR}"/conf.d zoneminder
+
+	# systemd unit file
+	systemd_dounit "${FILESDIR}"/zoneminder.service
+
+	cp "${FILESDIR}"/10_zoneminder.conf "${T}"/10_zoneminder.conf || die
+	sed -i "${T}"/10_zoneminder.conf -e "s:%ZM_WEBDIR%:${MY_ZM_WEBDIR}:g" || die
```

Comment:
This is done in `src_prepare`. You can also use `${S}`, but it doesn't matter much.

Reply:
will move and replace `${T}` with `${S}`